### ISSUE TITLE
Should your library join doublet ?

### ISSUE THREAD

**Issue by lpatiny** [cheminfo/multiplet-analysis#22]
Is it your library that should join the doublets if J are very similar to transform them to triplet, q, quint, hex, hept, ...

**Comment by djeanner** [comment]
Yes or no. It could (and probably should) but we would need a criterion for fusing J's (a delta below which we consider them as equal), but we could also argue that it should be done dowstream where some knowledge about the spins system allows to decide if indeed three coupling are equal or not. Finally, we could do both: return a "compressed" description including degenrecy.
Related, I want to include an option where the degeneracy is in the input: if we know something is a td (from the spin system) we would search directly for t and d in the multiplets (this is why I added the multiplicity.js in my project - I will need it to implement this).
You can add a Issue asking for generation of degeneracy, and propose a default criterion.
